x-pack/solutions/observability/plugins/observability/public/components/custom_threshold/custom_threshold_rule_expression.tsx

```diff
@@ -34,6 +36,7 @@ import {
 } from '@kbn/triggers-actions-ui-plugin/public';
 
 import { COMPARATORS } from '@kbn/alerting-comparators';
+import { SimpleSavedObject } from '@kbn/core/public';
```

Comment:
TODO: This is deprecated

Reply:
I removed [here](https://github.com/elastic/kibana/pull/209668/commits/003b4ceef7deb8703b0d2e562e6eccf97120b5b2)